content/intro-to-storybook/angular/pt/deploy.md

```diff
@@ -85,10 +85,12 @@ Seguida da seleção do repositório da lista de opções.
 
 ![Conexão Netlify para o repositorio](/intro-to-storybook/netlify-account-picker.png)
 
-É feita a configuração no Netlify ao selecionar-se o comando apropriado para executar no IC(CI na forma nativa) e qual a pasta de output. Como ramo, seleciona-se `master`. Pasta `storybook-static`. Comando `yarn build-storybook`.
+É feita a configuração no Netlify ao selecionar-se o comando apropriado para executar no IC(CI na forma nativa) e qual a pasta de output. Como ramo, seleciona-se `master`. Pasta `.storybook-static`. Comando `yarn build-storybook`.
 
 ![Configurações Netlify](/intro-to-storybook/netlify-settings.png)
 
+<div class="aside"><p>Caso o deploy no Netlify falhe, adicione a <a href="https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook">--quiet </a> flag ao seu comando <code>build-storybook</code>.</p></div>
```

Comment:
@WagnerMoreira remove the `seu` from here, leave it only as `adicione a flag ao comando.....`. To make it neutral and more approachable to both portuguese speaking audiences.

Reply:
@jonniebigodes  thanks! updated

Reply:
No problem.